src/main/res/values/strings.xml

```diff
@@ -375,7 +375,7 @@
     <string name="props_url">URL</string>
     <string name="props_num_comments">Number of Comments</string>
     <string name="props_body_markdown">Body (Raw Markdown)</string>
-    <string name="props_self_markdown">Self text (Raw Markdown)</string>
+    <string name="props_self_markdown">Self-text (Raw Markdown)</string>
```

Comment:
I don't think reddit hyphenate "self text" (applies throughout the diff)

Reply:
Pretty sure I checked, will find out.